api/lib/infrastructure/repositories/sessions/session-for-supervising-repository.js

```diff
@@ -1,11 +1,17 @@
 import { knex } from '../../../../db/knex-database-connection.js';
 import { NotFoundError } from '../../../domain/errors.js';
 import { CertificationCandidateForSupervising } from '../../../domain/models/CertificationCandidateForSupervising.js';
+import { CertificationChallengeLiveAlertStatus } from '../../../domain/models/CertificationChallengeLiveAlert.js';
 import { ComplementaryCertificationForSupervising } from '../../../domain/models/ComplementaryCertificationForSupervising.js';
 import { SessionForSupervising } from '../../../domain/read-models/SessionForSupervising.js';
 
 const get = async function (idSession) {
   const results = await knex
+    .with('ongoing-live-alerts', (qb) => {
+      qb.select('*')
```

Comment:
Pinaillage par habitude de pas mettre d'abbréviation même pour les trucs évidents

```suggestion
    .with('ongoing-live-alerts', (queryBuilder) => {
      queryBuilder.select('*')
```